### PR TITLE
Feature/claimant rep application

### PR DESCRIPTION
**Pull request by LahiruOshara**
### Change description ###

- claimant rep application related changes
- includes app creation, view and respond related changes

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: definitions/json/CaseEvent/CaseEvent.json [hunk 1]
```diff
@@ -1016,5 +1016,48 @@
     "SecurityClassification": "Public",
     "ShowEventNotes": "N",
     "ShowSummary": "N"
+  },
+  {
+    "CaseTypeID": "ET_Scotland",
+    "ID": "claimantTSE",
+    "Name": "Make an Application",
+    "Description": "Claimant Tell Something Else",
+    "PreConditionState(s)": "Submitted;Vetted;Rejected;Accepted",
```
Comment (hsjhita) [inline definitions/json/CaseEvent/CaseEvent.json:1025]: They should also be able to make applications when it's closed
```suggestion
    "PreConditionState(s)": "Submitted;Vetted;Rejected;Accepted;Closed",
```

File: definitions/json/CaseEventToFields/CaseEventToFields-WA.json [hunk 2]
```diff
@@ -169,5 +169,28 @@
...
+    "CaseEventID": "tseClaimantRepResponse",
+    "CaseFieldID": "genericTseApplicationCollection",
+    "FieldShowCondition": "claimantRepRespondSelectApplication=\"dummy\"",
+    "DisplayContext": "COMPLEX",
+    "PageID": 2,
+    "PageDisplayOrder": 2,
+    "PageFieldDisplayOrder": 1,
+    "ShowSummaryChangeOption": "N",
+    "PageColumnNumber": 1
+  }	  }
```
Comment (hsjhita) [inline definitions/json/CaseEventToFields/CaseEventToFields-WA.json:195]:
```suggestion
  }	
```

Comment (hsjhita) [reply definitions/json/CaseEventToFields/CaseEventToFields-WA.json:195]: Extra curly brace ^

File: definitions/json/CaseEventToFields/CaseEventToFields.json [hunk 3]
```diff
@@ -9040,5 +9040,747 @@
...
+  "CaseFieldID": "claimantTseRespNotAvailable",
+  "FieldShowCondition": "claimantRepRespondSelectApplication=\"dummy\"",
+  "DisplayContext": "READONLY",
+  "PageID": 1,
+  "PageDisplayOrder": 1,
+  "PageFieldDisplayOrder": 1,
+  "CallBackURLMidEvent": "${ET_COS_URL}/tseClaimantRepResponse/showError",
+  "PageColumnNumber": 1
+},
+  {
```
Comment (hsjhita) [inline definitions/json/CaseEventToFields/CaseEventToFields.json:9615]:
```suggestion
  },
  {
    "CaseTypeID": "ET_Scotland",
    "CaseEventID": "tseClaimantRepResponse",
    "CaseFieldID": "claimantTseRespNotAvailable",
    "FieldShowCondition": "claimantRepRespondSelectApplication=\"dummy\"",
    "DisplayContext": "READONLY",
    "PageID": 1,
    "PageDisplayOrder": 1,
    "PageFieldDisplayOrder": 1,
    "CallBackURLMidEvent": "${ET_COS_URL}/tseClaimantRepResponse/showError",
    "PageColumnNumber": 1
  },
  {
```

**Review by hsjhita — COMMENTED** [review]
See comments above

**Review by hsjhita — APPROVED** [review]

**Review by Mike-Brain — APPROVED** [review]

**Review by hsjhita — APPROVED** [review]

**Pull request merged**